### PR TITLE
Update ADOPTERS.md

### PR DESCRIPTION
**Pull request by jeremysf**
Love this software!

Signed-off-by: Jeremy Gordon <jeremy.gordon@gmail.com>

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -20,6 +20,7 @@
 - [Nordstrom](https://nordstrom.com/)
 - [Paybase](https://paybase.io/)
 - [PlayStudios Asia](https://www.playstudios.asia)
+- [Projector] (https://projector.com)
```
Comment (hawkw) [inline ADOPTERS.md:23]: Is this valid markdown? I think it needs to be 
```suggestion
- [Projector](https://projector.com)
```

**Comment by hawkw** [comment]
:raised_hands:

**Comment by alpeb** [comment]
@jeremysf would you mind doing another merge from master to have the CI tests pass? :pray:

**Comment by alpeb** [comment]
Merged in #4529

**Pull request closed without merging**